pkg/cmd/cli/describe/describer.go

```diff
@@ -363,17 +369,66 @@ func (d *ImageDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
-	return describeImage(image)
+	return describeImage(image, "")
 }
 
-func describeImage(image *imageapi.Image) (string, error) {
+func describeImage(image *imageapi.Image, imageName string) (string, error) {
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, image.ObjectMeta)
 		formatString(out, "Docker Image", image.DockerImageReference)
+		if len(imageName) > 0 {
+			formatString(out, "Image Name", imageName)
+		}
+		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
+		formatString(out, "Layer Size", units.HumanSize(float64(image.DockerImageMetadata.Size)))
```

Comment:
Any other comments then?

----- Original Message -----
> >  	return tabbedString(func(out *tabwriter.Writer) error {
> >  		formatMeta(out, image.ObjectMeta)
> >  		formatString(out, "Docker Image", image.DockerImageReference)
> > +		if len(imageName) > 0 {
> > +			formatString(out, "Image Name", imageName)
> > +		}
> > +		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
> > +		formatString(out, "Layer Size",
> > units.HumanSize(float64(image.DockerImageMetadata.Size)))
> 
> The v2, schemaVersion=1 manifest that is in Docker 1.6 does have a list of
> all the v1 layer metadata, which means we can iterate through them and sum
> the Sizes. I don't know if there's any guarantee that this will continue to
> function this way, but we can do it for now.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2356/files#r30638271